frontend-public/src/utils/environmentUtils.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * detect IE
+ * @returns version of IE or false, if browser is not Internet Explorer
+ */
+// This file is anticipated to have multiple exports
+// eslint-disable-next-line import/prefer-default-export
+export const detectIE = () => {
```

Comment:
Were you able to test this somehow? I would think with the current IE issue, that wouldn't be possible

Reply:
Haven't tested yet, need to deploy to dev to test the logic!

It doesn't show up for chrome or safari.